# Impact of sequencing technologies on long non-coding RNA computational identification

## Abstract

The correct annotation of non-coding RNAs, especially long non-coding RNAs (lncRNAs), is still an important challenge in genome analyses. One crucial issue in lncRNA transcript annotation is the transcriptome resource that supports lncRNA loci. Long-read technologies now bring the potential to improve the quality of transcriptome annotation. Consequently, long non-coding RNAs (lncRNA) are probably the most benefited class of transcripts that would have improved annotation using this novel technology. However, there is a gap regarding benchmarking studies that highlighted if the direct use of lncRNA predictors in long-reads makes more precise identification of these transcripts. Considering that these lncRNA tools were not trained with these reads, we want to address: how is the performance of these tools? Are they also able to efficiently identify lncRNAs? We could provide evidence of where and how to make potential better approaches for the lncRNA annotation by understanding these issues.

## File descriptions:

- [gencode_v21_intersection_by_tools_coding.csv.csv](https://github.com/alerpaschoal/Benchmarking-HTS-LncRNAs-Tools/raw/master/gencode_v21_intersection_by_tools_coding.csv.csv): Sequences classified as coding by the tools in GENCODE dataset version 21;
- [gencode_v38_intersection_by_tools_coding.csv](https://github.com/alerpaschoal/Benchmarking-HTS-LncRNAs-Tools/raw/master/gencode_v38_intersection_by_tools_coding.csv): Sequences classified as coding by the tools in GENCODE dataset version 38;

## Team

- Alisson Gaspar Chiquitto
- Lucas Ot√°vio Leme Silva
- Douglas Silva Domingues
- Alexandre Rossi Paschoal

Contact: paschoal@utfpr.edu.br
